package.json

```diff
@@ -59,4 +63,4 @@
         "vite-bundle-visualizer": "1.2.1",
         "vitest": "3.2.4"
     }
-}
\ No newline at end of file
+}
```

Comment:
Please install editorconfig (described in contributing.md) so you aren't adding whitespace changes

Reply:
`.editorconfig` has the following for common files: 

```
[*]
end_of_line = lf
insert_final_newline = true
indent_style = space
indent_size = 4
```

This is inserting the final line in the file so, I think it's OK.

Reply:
Ahh okay, it must have been off in the earlier file version. Thanks for checking and sorry for the confusion